packages/runtime/src/templates/ipx.ts

```diff
@@ -10,5 +10,6 @@ export const handler: Handler = createIPXHandler({
   domains,
   remotePatterns,
   responseHeaders,
+  localPrefix: '/_next/static/media/',
```

Comment:
We are noticing a breaking change as well. Is it possible that this is expecting images to be inside a `media` folder?